imaging/ml/ml_codelab/breast_density_auto_ml.ipynb

```diff
@@ -353,15 +353,15 @@
     "      if response['done'] == True and 'error' not in response:\n",
     "        success = True;\n",
     "      break\n",
-    "    time.sleep(30)\n",
+    "    time.sleep(60*5)\n",
```

Comment:
Can you change the methodr **wait_for_operation_completion** to take a variable sleep_time, which is set to 30seconds here and 5 minutes for the training portion? Please do this for both codelabs. Also, please move the:
 `path = os.path.join(AUTOML_API_URL, operation_name)
timeout = time.time() + 10*60 # Wait up to 10 minutes.
_ = wait_for_operation_completion(path, timeout)`

To be a code block of its own to keep the method independent.

Reply:
>Please do this for both codelabs.

`wait_for_operation_completion` is present only in breast_density_auto_ml.ipynb

Reply:
okay, just for auto_ml!

Reply:
done